stunnel/stunnel.ml

```diff
@@ -344,33 +344,29 @@ let connect
     | None -> () in
 	retry (fun () -> attempt_one_connect ?unique_id ?use_fork_exec_helper ?write_to_log _verify_cert extended_diagnosis host port) 5
 
-let sub_after i s =
-  let len = String.length s in
-    String.sub s i (len - i)
-
-let split_1 c s =
-  match String.split ~limit:1 c s with
-    | x :: _ -> x
-    | [] -> s
-
 let check_verify_error line =
-  match String.find_all "VERIFY ERROR: " line with
-      | p :: _ ->
-          begin
-            match String.find_all "error=" line with
-              | e :: _ ->
-                  raise
-                    (Stunnel_verify_error
-                       (split_1 ','
-                          (sub_after (e + String.length "error=") line)))
-              | [] ->
-                  raise (Stunnel_verify_error "")
-          end
-      | [] ->
-          ()
+  let sub_after i s =
```

Comment:
Would any of the substring functions from here already implement this?
http://erratique.ch/software/astring/doc/Astring.String.html#subs
http://erratique.ch/software/astring/doc/Astring.String.Sub.html